src/test/java/org/apache/mesos/testutils/OfferRequirementTestUtils.java

```diff
@@ -0,0 +1,41 @@
+package org.apache.mesos.testutils;
+
+import org.apache.mesos.Protos;
+import org.apache.mesos.offer.InvalidRequirementException;
+import org.apache.mesos.offer.OfferRequirement;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * This class provides utility methods for tests concerned with OfferRequirements.
+ */
+public class OfferRequirementTestUtils {
+    public static OfferRequirement getOfferRequirement(Protos.Resource resource) throws InvalidRequirementException {
+        return getOfferRequirement(Arrays.asList(resource));
```

Comment:
Yes, to keep things DRY.  It was being reimplemented a few times in a few different test classes.